packages/datepicker/src/css/index.ts

```diff
@@ -1,229 +1,179 @@
 import {
-  colorsStatus,
-  colorsBackgroundDark,
-  colorsBackgroundLight,
+  colorsWhite,
   colorsTextIcon,
-  colorsBorder,
-  colorsPrimaryAction,
-  layout,
+  colorsBlue,
   type,
-  motion
+  motion,
+  layout
 } from '@pluralsight/ps-design-system-core'
-import { widths as iconWidths } from '@pluralsight/ps-design-system-icon'
-import { names as themeNames } from '@pluralsight/ps-design-system-theme'
-
-import * as vars from '../vars'
-
-const dayDimensions = {
-  height: '32px',
-  width: '32px',
-  margin: `0 2px ${layout.spacingXSmall} 0`,
-  border: '3px solid transparent'
-}
-
 export default {
-  '.psds-date-picker': {
-    display: 'inline-block'
-  },
-  '.psds-date-picker--disabled': {
-    opacity: 0.5
-  },
+  '@keyframes psds-calendar__keyframes__forward': {
```

Comment:
Seems worth calling out that the `keyframes__` prefix is a new pattern. I like it because it groups keyframe declarations separate from the other selectors, which we haven't necessarily done in other components. Is this something we should adopt broadly?

Reply:
I think so 

> A few questions inline and a few things that seem like things to address:
> 
> * I'd squash the "--wip-- [skip ci]" commits so we don't keep those for the git history.
> * Calendar selection will select a date in stories, but input typing will not select a date. I believe it should since that was the old ux, but perhaps I'm missing something.
> 
> ![2021-01-19 14 52 59](https://user-images.githubusercontent.com/552044/105097970-a49f4f00-5a66-11eb-94e8-c63384c56951.gif)
> 
> * Input editing for a date already selected does not seem to work properly. Backspace and typing corrupts the field value. The moment of date selection is off or never happens afaik.
> 
> ![2021-01-19 14 51 13](https://user-images.githubusercontent.com/552044/105098111-d4e6ed80-5a66-11eb-8606-49bfc5058a4f.gif)
> 
> Overall, a great bunch of work. Thanks for working to make this nice. I think it just needs a few use cases polished out.
> 
> I also noticed your comments in the PR about what was yet needed. Are you planning to do that in a future PR after a merge? Or are you going to refactor and/or add what's missing in this same PR and then want another round of reviews?

Though I fixed the input errors. Hmm let me go look again. As for the key frames pattern yep I think we should adopt it. I got a bunch of doc typos and suggestions from Dane to implement.